security/config.py

```diff
@@ -44,6 +44,8 @@
     'BACKUP_STORAGE_PATH': None,
     'DEBUG_TOOLBAR': False,
     'SHOW_DEBUG_TOOLBAR': False,
+    'CELERY_HEALTH_CHECK_DEFAULT_QUEUE': 'default',
+    'CELERY_HEALTH_CHECK_MAX_TASKS_COUNT': 50,
```

Comment:
Je otázka, co je tady "sane default".

Reply:
IMO highly depends on project... but this looks ok to me.